### DISCUSSION TITLE
How to replace rem with vw？

### DISCUSSION THREAD

**Discussion by ShaneLee-9** [tailwindlabs/tailwindcss#11753] [Help]
I want to replace all units with vw at once.

In our project, we use `vw` as a means of adapting to mobile. For example, in a 750px design draft, I have a div with a width of 100px, and I want to write `w-100 ` or `w-[100px]` directly when using tailwindcss, and the build outputs `width: 13.3333vw`. Since I want to apply this solution to all rules (etc. `height`, `margin`, `padding`), I can't use [matchUtilities](https://tailwindcss.com/docs/plugins#dynamic-utilities).

I've done some searching, but unfortunately I didn't find a solution.

**Comment by wongjn** [comment]
You could consider customizing [the `spacing` values](https://tailwindcss.com/docs/customizing-spacing). Many of the utilities, such as `width`, `margin`, `padding`, etc derive their values from this central dictionary.
